jobs/yolov5-web-service/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM pytorch/pytorch
+
+WORKDIR /workspace
+
+ADD . /workspace
+
+RUN pip install -r requirements.txt
+
+CMD [ "python" , "/workspace/app.py" , "--host=0.0.0.0" ]
```

Comment:
Finally you call your script directly with the python `__main__` and you already specify `host='0.0.0.0'` in it.
So I think you don't need the `--host=0.0.0.0` in the `CMD` anymore.